### ISSUE TITLE
Automatically use correct IAM User roles based on logged in user

### ISSUE THREAD

**Issue by jasampson** [tilfinltd/aws-extend-switch-roles#73]
First, some background on my particular scenario.  I have one AWS account ID and multiple IAM Users with different roles.  To switch IAM Users, I obviously have to log out of the AWS console and log back in with a different IAM user name.

The problem I'm having is that the plugin doesn't seem to show the correct roles list for the currently logged in IAM user.  It seems to only use the IAM roles from whichever IAM user is listed last in the config.

Is there currently any way for the plugin to simply match my IAM user with the one in the config and intelligently show only roles that apply to that particular IAM user? 

Here is what my config looks like:
```
[IAM-User-1]
aws_account_id = exampleaccount

[IAM-User-2]
aws_account_id = exampleaccount

[ExampleAcct1]
role_arn = arn:aws:iam::123456789012:role/AssumeRoleTest
source_profile = IAM-User-1

[ExampleAcct2]
role_arn = arn:aws:iam::123456789012:role/AssumeRoleOther
source_profile = IAM-User-2
```

**Comment by tilfin** [comment]
This extension does not expect to log in as multiple users of the same AWS account. Which is the difference of two IAM users, (username and role) or role only?

**Comment by tilfin** [comment]
This issue will be satisfied by #129 .

**Comment by tilfin** [comment]
@jasampson Is this issue solved by the feature 'base account with role_name' released at latest version 2.2.0?

**Comment by jasampson** [comment]
Confirmed, this issue is now resolved.  Thank you!
